### ISSUE TITLE
jQuery document ready

### ISSUE THREAD

**Issue by alex9pro** [michalsnik/aos#12]
When i init AOS in jQuery document ready function AOS isn't initialized
`
$(document).on('ready', function(){
  AOS.init();
});
`

**Comment by michalsnik** [comment]
Hi, It's because AOS listens for `DOMContentLoaded` event and when you init AOS on document ready, the event is not triggered again, so AOS is not going to be initialized.

Put `AOS.init()` outside document ready and it'll work fine.

You can also set different start event:
```
AOS.init({
   startEvent: 'yourEvent'
});
```
and now you can trigger `yourEvent` on document whenever you want and AOS will be initialized.

For example:
```
AOS.init({
  startEvent: 'yourEvent'
});

$(document).on('ready', function() {
  $(document).trigger('yourEvent');
});
```

**Comment by michalsnik** [comment]
Though, it's worth making it possible to initialize on document ready as well. It'll probably save some headaches with settings this up. I'll consider this in future releases.
Thanks for your contribution @alex9pro !

**Comment by michalsnik** [comment]
Good news @alex9pro, I fixed it while making AOS compatible with different module systems (#14), as this change was necessary to make AOS work as AMD module.

**Comment by alex9pro** [comment]
@michalsnik, that is good news, thank you

**Comment by michalsnik** [comment]
It will be available in version 1.1.0
